src/utils/normalize-otherAttribute-contact.ts

```diff
@@ -0,0 +1,52 @@
+import cloneDeep from 'lodash/cloneDeep';
+import forEach from 'lodash/forEach';
+import { denormalize } from '../normalize';
+import { ContactInputRequest } from '../normalize/entities';
+
+export function createContactBody(data: any) {
+	const { attributes, ...rest } = data;
+	const contactAttrs = <Object[]>[];
+
+	forEach(attributes, (val, key) =>
+		key !== 'other'
+			? contactAttrs.push({
+					name: key,
+					[key === 'image' ? 'aid' : 'content']: val
+			  })
+			: forEach(val, (otherValue: any, index: any) =>
+					contactAttrs.push({
+						name: 'custom' + (index + 1),
+						_content: otherValue
+					})
+			  )
+	);
+	return {
+		cn: {
+			...denormalize(ContactInputRequest)({
+				...rest,
+				attributes: contactAttrs
+			})
+		}
+	};
+}
+
+export function normalizeOtherAttr(data: any) {
+	return data.map((contact: any) => {
+		let other: any = [];
+		let contactCopy = cloneDeep(contact);
```

Comment:
I don't think we need to `cloneDeep` the contact here, using `...contact` should be OK.

Reply:
Removed.